src/wp-includes/functions.php

```diff
@@ -8061,6 +8061,91 @@ function wp_unique_id_from_values( array $data, string $prefix = '' ): string {
 	return $prefix . $hash;
 }
 
+/**
+ * Retrieves cached query data if valid and unchanged.
+ *
+ * @param string $cache_key The cache key used for storage and retrieval.
+ * @param string $group The cache group used for organizing data.
+ * @param string $last_changed The timestamp of the last modification to the cache group.
+ * @return mixed|false The cached data if valid, or false if the cache does not exist or is outdated.
+ */
+function wp_cache_get_query_data( $cache_key, $group, $last_changed ) {
+	$cache = wp_cache_get( $cache_key, $group );
+
+	if ( ! is_array( $cache ) ) {
+		return false;
+	}
+
+	if ( ! isset( $cache['last_changed'], $cache['data'] ) || $last_changed !== $cache['last_changed'] ) {
```

Comment:
I would recommend
```suggestion
	if ( ( ! isset( $cache['last_changed'] ) && ! isset( $cache['data'] ) ) || $last_changed !== $cache['last_changed'] ) {
```
The behaviour of multiple parameters to `isset` is non-obvious, this makes it more explicit.

Reply:
@rmccue Is this a style thing or is the a code reason you want this change?

Reply:
I also recently saw a PR with `isset` with two variables and while assumed it only returned true if both were set, it wasn't immediately obvious to me because I hadn't actually seen isset used with two variables before (so I had to check the docs). I think Ryan is just saying its a bit clearer when reading the code to use two isset checks. Personally I like the way you wrote it, its more succinct and developers reviewing the code will learn something new like I did.

Reply:
Single `isset()` is fine IMO, but I see Ryan's point.

Reply:
Fixed.